lib/containers/container_images.pm

```diff
@@ -119,12 +119,13 @@ sub test_opensuse_based_image {
     $version = 'Tumbleweed' if ($version =~ /^Staging:/);
 
     if ($image_id =~ 'sles') {
-        my $pretty_version = $version =~ s/-SP/ SP/r;
-        my $betaversion    = get_var('BETA') ? '\s\([^)]+\)' : '';
-        record_info "Validating", "Validating That $image has $pretty_version on /etc/os-release";
-        validate_script_output("$runtime container run --entrypoint '/bin/bash' --rm $image -c 'grep PRETTY_NAME /etc/os-release' | cut -d= -f2",
-            sub { /"SUSE Linux Enterprise Server ${pretty_version}${betaversion}"/ });
-
+        unless ($host_id =~ 'suse-microos') {
```

Comment:
Should be moved into the `if ($host_id =~ 'sles')` block, which is actually a more fitting condition